server/src/pages/shot/editor.js

```diff
@@ -140,3 +146,54 @@ exports.Editor = class Editor extends React.Component {
     this.drawContext.stroke();
   }
 }
+
+class ColorPicker extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      picker: 'inactive',
+      color: '#000'
+    };
+  }
+
+  render() {
+    let colorBoard;
+    if (this.state.picker == 'active') {
+      colorBoard = <div className="color-board">
+        <div className="row">
+          <div className="swatch" title="White" style={{backgroundColor: "#FFF", border: "1px solid #000"}} onClick={this.onClickSwatch.bind(this)}></div>
+          <div className="swatch" title="Black"style={{backgroundColor: "#000"}} onClick={this.onClickSwatch.bind(this)}></div>
+          <div className="swatch" title="Alizarin"style={{backgroundColor: "#E74C3C"}} onClick={this.onClickSwatch.bind(this)}></div>
+        </div>
+        <div className="row">
+          <div className="swatch" title="Emerald" style={{backgroundColor: "#2ECC71"}} onClick={this.onClickSwatch.bind(this)}></div>
+          <div className="swatch" title="Peter River" style={{backgroundColor: "#3498DB"}} onClick={this.onClickSwatch.bind(this)}></div>
+          <div className="swatch" title="Yellow" style={{backgroundColor: "#FF0"}} onClick={this.onClickSwatch.bind(this)}></div>
+        </div>
+        <div className="row">
+          <div className="swatch" title="Wisteria" style={{backgroundColor: "#8E44AD"}} onClick={this.onClickSwatch.bind(this)}></div>
+          <div className="swatch" title="Turquoise" style={{backgroundColor: "#1ABC9C"}} onClick={this.onClickSwatch.bind(this)}></div>
+          <div className="swatch" title="Wet Asphalt" style={{backgroundColor: "#34495E"}} onClick={this.onClickSwatch.bind(this)}></div>
```

Comment:
Hah, I was kind of kidding about color names, but this is great.  Not sure how localization is going to work for something like this.  Maybe @flodolo can enlighten us.

Reply:
Oh boy, they're going to have fun… (I need to google Wisteria)

Reply:
@flodolo @chenba We can stick with blues, greens and yellows!